apps/editing-toolkit/package.json

```diff
@@ -114,7 +114,7 @@
 		"@automattic/calypso-eslint-overrides": "workspace:^",
 		"@testing-library/jest-dom": "^5.15.1",
 		"@testing-library/react": "^12.1.2",
-		"@types/node": "^15.0.2",
```

Comment:
I don't believe we can allow ETK to update yet. (context in previous PRs on the same issue. I guess I should try to disable renovate for this instance of ETK)